app/components/views/GetStartedPage/DaemonLoading/Form.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import Header from "../../../Header";
+import LinearProgress from "material-ui/LinearProgress";
+import KeyBlueButton from "../../../KeyBlueButton";
+import "../../../../style/GetStarted.less";
+import ReactToolTip from "react-tooltip";
+
+const DaemonLoadingFormHeader = ({
+  startupError,
+  getDaemonStarted,
+  getCurrentBlockCount,
+}) => (
+  <Header getStarted
+    headerTitleOverview="Setting up Decrediton"
+    headerMetaOverview={getDaemonStarted ? getCurrentBlockCount == null ? "Daemon started, waiting for rpc..." : "Blockchain syncing" : "Starting daemon"}
```

Comment:
I would capitalize "rpc" here as it's an acronym, and to be consistent with the other startup message.